docs/recipes/shared-workers.md

```diff
@@ -34,6 +34,25 @@ const shared = registerSharedWorker({
 
 Within a test process you can only register one worker for each `filename`. Filenames are compared as-is, without normalization. If you call `registerSharedWorker()` a second time, the same worker instance is returned.
 
+If for some reason you want to load the same file as multiple different workers, you can append a unique hash to the end of the filename:
+
+```js
+import crypto from 'crypto';
+import {registerSharedWorker} from 'ava/plugin';
+
+const randomKey = crypto.randomBytes(20).toString('hex');
```

Comment:
It can't be random, then each test worker will end up loading a different shared worker… so it won't be shared.

Reply:
Whoops, my bad. 😅